help_to_heat/frontdoor/views.py

```diff
@@ -718,9 +718,9 @@ def save_post_data(self, data, session_id, page_name):
 @register_page(address_select_page)
 class AddressSelectView(PageView):
     def build_extra_context(self, request, session_id, page_name, data, is_change_page):
-        data = interface.api.session.get_answer(session_id, address_page)
-        building_name_or_number = data[address_building_name_or_number_field]
-        postcode = data[address_postcode_field]
+        address_data = interface.api.session.get_answer(session_id, address_page)
+        building_name_or_number = address_data[address_building_name_or_number_field].strip()
```

Comment:
could possibly go more expansive on this (ie trimming the address & postcode before saving to the session), though wondered whether this would be worthwhile overall. depends on whether we want to trim the input before sending to EPC API too (though testing shows this is unnecessary)

Reply:
Would probably vote for doing that tbh (trimming after input), just so it's not something we ever have to worry about.

Reply:
have reworked to do this now